common/src/main/java/com/microsoft/identity/common/adal/internal/AuthenticationConstants.java

```diff
@@ -1427,6 +1442,11 @@
          * BrokerContentProvider URI code constant for BrokerApi-to-Broker updateBrt request.
          */
         public static final int BROKER_API_UPDATE_BRT_CODE = 12;
+
+        /**
+         * BrokerContentProvider URI code constant for MSAL-to-Broker generateSHR request.
```

Comment:
@rpdome So this value is completely arbitrary? Can I 'claim' `13`?

Reply:
yep